R/training_window.R

```diff
@@ -0,0 +1,109 @@
+#' Limits the size of the training window to the most recent observations
+#'
+#' `step_training_window` creates a *specification* of a recipe step that
+#'   limit the size of the training window to the `nrec` most recent
+#'   observations in `time_value` per location from `geo_value`.
+#'
```

Comment:
We probably need some sort of warning to indicate that `step_epi_lead()` and `step_epi_lag()` should come AFTER any filtering.

Reply:
Added a note for this in `@details` part of the documentation.